### DISCUSSION TITLE
Formatting Uniq result as string

### DISCUSSION THREAD

**Discussion by toog** [fbaligand/kibana-enhanced-table#264] [Q&A] [answered]
Hello

![image](https://user-images.githubusercontent.com/40697794/160377313-90156b37-18ab-4908-b0f4-a83cca090af2.png)

I want to have the uniq result without formatting like an array of string, I try to use the replace function to remove braket and quote but that failed.

Because I think uniq result is an array, how to transform it to string ?

![image](https://user-images.githubusercontent.com/40697794/160377940-1814438b-ac8f-427e-a02b-6c9536118033.png)

Have you a solution ?

**Comment by fbaligand** [comment, marked as answer]
Hi,

To do that, you can use `join` function.
Like that: `join(", ", uniq(col0))`

More globally, you can find all basic `expr-eval` functions here: 
https://github.com/silentmatt/expr-eval#pre-defined-functions

**Comment by toog** [comment]
thanks !
